lib/tasks/check_changelog.rake

```diff
@@ -17,8 +17,13 @@
 require 'rake'
 
 namespace "check" do
-  desc "Checking for new IDs (bugzilla,fate,...)  in *.changes file"
+  desc "Checking for new IDs (bugzilla,fate,...) in *.changes file"
   task :changelog do
+    if obs_sr_project.nil?
+      $stderr.puts "Skipping the check because OBS submit request project "\
+        "is not defined (obs_sr_project)."
```

Comment:
well, it is different meaning. if you use `if verbose` then it is written unless --quiet option is passed. If you use `if verbose == true` then it is written only if you use `--verbose`